### DISCUSSION TITLE
Repos for default layout extensions (Table, Calendar, Cards, etc..)

### DISCUSSION THREAD

**Discussion by divinepersuasionstudio** [directus/directus#18296] [Feature Requests]
### Summary

It seems like many of us are in need of using one of the default layouts and making changes according to our needs. There is a lot of overhead built into these default layouts and it is very difficult to write from scratch.

### Motivation

Easier to create custom layout extensions without starting from scratch.

### Detailed Design

It can be as simple as providing a repo for these default layouts where we can clone and use as the beginning point of our custom extension.

### Must Have:
- default layouts repo

### Should Have:
- documentation examples

### Could Have:
- <3

### Won't Have:
- the current way the default layouts are not built as custom extensions

### Drawbacks

none I can think of

### Alternatives

haven't found any

### Adoption Strategy

it doesn't have to be a change to the main directus repo at all. It can be as simple as providing us repos for the default layout extensions (Table, Calendar, Cards, etc) which will give us a good place to start.

### Unresolved Questions

<3

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
